apps/src/blockTooltips/DropletAutocompleteParameterTooltipManager.js

```diff
@@ -95,7 +95,9 @@ DropletAutocompleteParameterTooltipManager.prototype.showParamDropdownIfNeeded_
     this.dropletTooltipManager.codeFunctions,
     this.autocompletePaletteApisOnly).forEach(function (block) {
       if (!block.dropdown ||
-          (block.func !== paramInfo.funcName && block.func !== paramInfo.fullFuncName)) {
+          (block.func !== paramInfo.funcName &&
+          block.modeOptionName !== paramInfo.funcName &&
+          block.func !== paramInfo.fullFuncName)) {
```

Comment:
This made the dropdown work in text mode.  @cpirich I think this also fixes a bug with some of the dropdowns you added for color methods (setColorEach, etc).